extensions/vscode/webviews/homeView/src/components/views/Credentials.vue

```diff
@@ -3,9 +3,12 @@
     title="Credentials"
     data-automation="credentials"
     :actions="sectionActions"
+    :codicon="home.credentialsAlert ? `codicon-alert` : ``"
```

Comment:
I'm not sure the having no Credentials should be an alert. This makes it so the very first experience, without a Deployment, shows an alert to the user.

![CleanShot 2024-09-26 at 14 37 22@2x](https://github.com/user-attachments/assets/c579f1f6-2a91-4fbd-8c3d-5e2b81dedf1e)

If we are in the
> A Credential for the Deployment's server URL was not found

state then I think that is valid.